ogre/include/ignition/rendering/ogre/detail/Export.hh

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <gz/rendering/ogre/Export.hh>
```

Comment:
This is missing `detail` 🧐 

But in any case we shouldn't need any `ignition/**/detail` files

Reply:
Now no longer created in https://github.com/gazebosim/gz-rendering/pull/618/commits/a4191f722debd816f291558460a23751699cda8e